### ISSUE TITLE
some screen reader accessibility bugs

### ISSUE THREAD

**Issue by RealAmethyst** [fluxerapp/fluxer#504]
### Summary

When launching fluxer on my windows PC my screen reader found two sets of minimize, maximize and close buttons related to the window, after logging in, this got fixed. When creating an account there are two check boxes that lets you agree to the user agreement thing, checking one also checks the other, but there is no reason for two. The direct messages tuggle button also has a submenu element for some reason, that is the same with favorites. There are submenus that aren't labeled, when tabbing after the find communities submenu there are two submenus with no label. Also one right under direct messages. (I say under but it's right after if you use tab. I am fully blind so I don't know how things are visually)
The user settings button seems to be a submenu as well, with no label. I really want to use this application going forward and I hope this issue can be a good place for accessibility issues with screen readers to get tracked.

### Steps to reproduce

Use any screen reader, Narrator or NVDA on Windows, VoiceOver on Mac or Orca on Linux, navigate around using the keyboard and you will see what I mean.

### Environment (optional)

Windows 11 Desktop client

### Checks

- [x] I searched for existing issues and didn't find a duplicate.

**Comment by RealAmethyst** [comment]
Just wanted to ad to this that it seems as there are more submenu elements when in a DM chat. I have nothing against submenus, but there are a lot of them where they don't make sense.

**Comment by RealAmethyst** [comment]
Sorry to ad to this even more and feel free to just tell me to create new issues for each thing that shows up, but the quick switches doesn't let me select stuff when inside the edit field. So let me explain a little more, when using quick switcher in something like Discord, I can start typing and results will show up in the edit field, meaning my screen reader speaks automaticaly. I can just use up or down errows to hear something like "general, text channel, server name" or "server name" or "DM name" Clicking enter then takes me to that without ever having to leave the edit field. In Fluxer this task is much more difficult to do, as I have to tab out of the edit field and try to find what I'm looking for.

**Comment by Kamalaja** [comment]
Bulk closed due to Fluxer V2 - If this issue is still present, please open a new issue.
